### DISCUSSION TITLE
Weather sensor data does not match Weather app

### DISCUSSION THREAD

**Discussion by zdunex25** [zepp-health/zeppos-forum#141] [Developer Feedback]
I recently released a watchface that is fully based on sensors and text widgets, however there is a problem when it comes to weather data sync.
[High, low and index ](https://docs.zepp.com/docs/watchface/api/hmSensor/sensorId/WEATHER/#forecastdataitem-object) never match to built-in Weather app, users are confused to see different temps and conditions because of it.

I know there is _hmUI.data_type.WEATHER_LOW_ etc, but you cannot pass it's data through _hmUI.widget.TEXT_.

@MyoungXUE the watchface i am talking about designed for gtr4 if you would like to see for yourself: [row_gtr4.zip](https://github.com/zepp-health/zeppos-forum/files/11242249/row_gtr4.zip)

![row](https://user-images.githubusercontent.com/1674725/232309570-3ba4d936-e68b-4272-b3ac-ec57d201a73a.gif)

**Comment by MyoungXUE** [comment]
Thank you for your feedback, I will feedback this issue to the relevant colleagues.

**Comment by MyoungXUE** [comment]
Hello, you can try the following solution first.

There is also a control called 'hmUI.widget.TEXT_FONT', compared with 'hmUI.widget.TEXT' control, has an additional attribute `type`, the value can be set 'hmUI.data_type.WEATHER_LOW' etc.

We are adding controls to the document, so we can try them out before we are done.
